.github/workflows/ci.yml

```diff
@@ -198,6 +198,16 @@ jobs:
       - name: "Check we're not using a -dev version of the embedded compiler"
         run: jq -r '.["compiler-version"]' package.json | grep -qv -- '-dev$'
 
+      - name: Publish optional dependencies
+        run: |
+          cd npm/linux-arm64 && npm publish && cd ../..
+          cd npm/linux-ia32 && npm publish && cd ../..
+          cd npm/linux-x64 && npm publish && cd ../..
+          cd npm/macos-arm64 && npm publish && cd ../..
+          cd npm/macos-x64 && npm publish && cd ../..
+          cd npm/windows-ia32 && npm publish && cd ../..
+          cd npm/windows-x64 && npm publish && cd ../..
```

Comment:
Rather than putting a bunch of publishing logic in the individual `package.json` files, I think it would be better to run it here even if it makes these invocations more complex.